cardano-launcher/app/Main.hs

```diff
@@ -136,7 +123,7 @@ data LauncherExceptions
     = CannotGenerateTLS
     | LauncherOptionsError Text
     | WorkingDirectoryDoesNotExist FilePath
-    deriving (Eq)
+    | FailedToGenerateTLS TLSError
```

Comment:
The `Eq` can be added here, I think. It can usually be useful to have `Eq` on types if possible.